packages/@aws-cdk/aws-scheduler-targets-alpha/test/integ.schedule.ts

```diff
@@ -0,0 +1,90 @@
+import * as scheduler from '@aws-cdk/aws-scheduler-alpha';
+import { ExpectedResult, IntegTest } from '@aws-cdk/integ-tests-alpha';
+import * as cdk from 'aws-cdk-lib';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import { LambdaInvoke } from '../lib';
+
+/*
+ * Stack verification steps:
+ * The lambda MyLambda is triggered after 5 minutes which adds a Tag:
+ *   Key: OutputValue
+ *   Value: base64 JSON string of the input event
+ * The assertion checks if it is the correct value
+ */
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-schedule');
+
+const functionName = 'MyTestFunc';
+const functionArn = stack.formatArn({
+  service: 'lambda',
+  resource: 'function',
+  resourceName: functionName,
+  arnFormat: cdk.ArnFormat.COLON_RESOURCE_NAME,
+});
+const payload = 'test';
+
+const func = new lambda.Function(stack, 'MyLambda', {
+  code: new lambda.InlineCode(`
```

Comment:
make this a `lambda.Asset` please. Motivation is that it is very hard to identify and make maintenance updates to this code if its inline.

Reply:
^